evalml/demos/weather.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+import pandas as pd
+
+from evalml.utils import infer_feature_types
+
+
+def load_weather():
```

Comment:
This is covered by our docs. I think we'll eventually change this code once we upload the dataset to s3 so I think it's ok to not add a unit test for now.